src/public/inventory.ts

```diff
@@ -133,5 +133,32 @@ export const inventoryPublicHandlers: PublicNatsHandler[] = [
         {
             queue: SERVICE_NAME
         }
+    ],
+    [
+        "PUT",
+        "inventory.*",
```

Comment:
the `.*` here means that we expect to obtain a `url parameter`, most likely the `inventory_item_id`, but we're not using it in the handler. How does this handler get the `inventory_item_id` that the `transfer_inventory_item` handler requires?

Reply:
I changed PUT to PATCH, and the handler now uses the inventory_item_id submitted by the user in the url as a parameter

Reply:
thanks that should work. Could you also make sure that both the public handler and private handler are added to a queue group? See the other public handlers on how to configure a queue group.
Some NATS doc here about queue groups:
- https://docs.nats.io/nats-concepts/queue
- https://github.com/nats-io/nats.js/#queue-groups

Reply:
looks good, thanks